app/build-js.sh

```diff
@@ -20,11 +20,24 @@ if ! which npm 1>/dev/null 2>&1; then
     exit 1
 fi
 
+# Add navitron & its dependencies to the app bundle
+pushd ../node_modules/navitron
```

Comment:
@jeremywiebe: To my understanding, in this case `$MYPATH` refers to `scaffold/app`.

Since there is already an initial `pushd $MYPATH`, and `pushd ../node_modules/navitron` is already relative to $MYPATH... did you mean you want:

`pushd $MYPATH/../node_modules/navitron`?

Or we could even have a variable called `ROOT` and have that point to `scaffold/` instead and base everything off of the root of the project rather than the path of the script.